Fix code regression

ihhub opened the pull request:
relates to #733

ihhub commented:
Hi @idshibanov , I reverted some of your changes as we introduced regression in our code :(

idshibanov commented:
> Hi @idshibanov , I reverted some of your changes as we introduced regression in our code :(

Got it, thanks for catching that.

ihhub commented:
> > Hi @idshibanov , I reverted some of your changes as we introduced regression in our code :(
> 
> Got it, thanks for catching that.

No worries, I am not blaming you :) It happens
